### ISSUE TITLE
[BUG] Artwork not showing when selecting context menu while using AF 1 with newest TMDB Helper versions.

### ISSUE THREAD

**Issue by woodside007** [jurialmunkey/plugin.video.themoviedb.helper#1290]
### Describe the bug

I know you retired Arctic Fuse 1 but I still prefer it over Arctic Fuse 2. I seen that you have pushed a newer version of TMDB Helper to the Kodi repo recently and have a couple of questions before I upgrade to the newest version that was recently pushed to the Kodi repo.

I reverted back to using the Kodi repo version 5.2.25 of TMDB Helper because in the newer versions the artwork doesn't show up when selecting the context menu while using Arctic Fuse 1. It lost the ability to show artwork due to recent changes in the code. The poster and landscape images do not show anymore using the newer versions of TMDB Helper when selecting the context menu on a show or movie.

If I install the newly pushed version 5.4.16 that is now on the Kodi repo while still using Arctic Fuse 1 am I going to lose the artwork showing on the context menu again? If so can you fix Arctic Fuse 1 so that it is compatible with the newest versions of TMDB Helper or make TMDB Helper backward compatible with Arctic Fuse 1? I really want to keep using Arctic Fuse 1 and I don't really like the changes being made on the modded version that is out there. Thanks.

### Steps To Reproduce

Use Arctic Fuse 1 with newer version of TMDB Helper. Select context menu on a movie or show. See emptiness where artwork should be.

### Checklist

- [X] I have searched the issues and this bug has not been reported.
- [X] I have updated TMDbHelper to the most recent version.
- [X] I have described the bug and included steps to reproduce the issue
- [ ] I have uploaded my debug log.

**Comment by jurialmunkey** [comment]
Skin issue. I think this was fixed in the mod version.

EDIT - sorry I saw you said you don't like the mod version. Only option then is to use an old version of TMDbhelper.

I'm not interested in working on AF1 further. It isn't even on my repo anymore. 

You could ask in the mod thread on the forum about how to fix for AF1 without using the full mod.

**Comment by woodside007** [comment]
> You could ask in the mod thread on the forum about how to fix for AF1 without using the full mod.

Yeah, I did ask a few weeks ago on the mod thread if he would share what he did to fix it but never got a reply. Seen that he was going to take a break for the holidays so I figured I would just wait till he was active again to ask once more. Figured I would just use the Kodi repo version until then but then that got updated as well. I then decided to try and give you a shout but if you're not interested in AF 1 any further I suppose I'm just out of luck. Cheers.

**Comment by jurialmunkey** [comment]
Most likely you would just need this file: https://github.com/jurialmunkey/skin.arctic.fuse.2/blob/omega/extras/tmdbhelper/baseitem.json

copied to `extras/tmdbhelper/`

**Comment by woodside007** [comment]
> Most likely you would just need this file: https://github.com/jurialmunkey/skin.arctic.fuse.2/blob/omega/extras/tmdbhelper/baseitem.json
> 
> copied to `extras/tmdbhelper/`

Copied it over but still no artwork in context menu. I guess there are other changes needed also. I tried looking at the github code changes for the modded version but hard to follow because he labels all his commits as version bumps. Hard to follow where the changes were made for that specific issue.

**Comment by OfficerKD637** [comment]
@woodside007 Hey!
Apologies for the late reply! Hope you've been enjoying the holidays! I missed your message on the forum before I went on a break, so I'm really sorry about that!

Once you add the baseitem.json file that Jurial posted above, you'll need to make the following changes:

1. Add `<onload>SetProperty(TMDbHelper.ContextMenu,True)</onload>` to the `Custom_1170_Dialog_HomeMenu.xml` file here ([line 7](https://github.com/jurialmunkey/skin.arctic.fuse/blob/6a2c9d463a1b1ae4e38eeddf2673c003f321bf5f/1080i/Custom_1170_Dialog_HomeMenu.xml#L7))

2. For the context menu, you'll need to edit the 4 variables (`Image_ContextPoster`, `Image_ContextFanart`, `Image_ContextClearLogo` and `Image_ContextIcon`) in the `Includes_Images.xml` file ([here](https://github.com/jurialmunkey/skin.arctic.fuse/blob/6a2c9d463a1b1ae4e38eeddf2673c003f321bf5f/1080i/Includes_Images.xml#L123-L141))

Replace those with the following:
```
<variable name="Image_ContextPoster">
        <value condition="!String.IsEmpty(Container(99950).ListItem.Property(base_poster))">$INFO[Container(99950).ListItem.Property(base_poster)]</value>
        <value condition="!String.IsEmpty(ListItem.Art(poster))">$INFO[ListItem.Art(poster)]</value>
    </variable>

    <variable name="Image_ContextFanart">
        <value condition="!String.IsEmpty(Container(99950).ListItem.Property(base_fanart))">$INFO[Container(99950).ListItem.Property(base_fanart)]</value>
        <value condition="!String.IsEmpty(ListItem.Art(fanart))">$INFO[ListItem.Art(fanart)]</value>
    </variable>

    <variable name="Image_ContextClearLogo">
        <value condition="!String.IsEmpty(Container(99950).ListItem.Property(base_clearlogo))">$INFO[Container(99950).ListItem.Property(base_clearlogo)]</value>
        <value condition="!String.IsEmpty(ListItem.Art(clearlogo))">$INFO[ListItem.Art(clearlogo)]</value>
    </variable>

    <variable name="Image_ContextIcon">
        <value condition="!String.IsEmpty(Container(99950).ListItem.Property(base_icon))">$INFO[Container(99950).ListItem.Property(base_icon)]</value>
        <value condition="!String.IsEmpty(ListItem.Art(icon))">$INFO[ListItem.Art(icon)]</value>
    </variable>
```

These should hopefully resolve those bugs! Feel free to reach out on the forum if you have any more questions.

As a side note (and a bit off topic haha), I know you mentioned you don't like my mod so please don't hesitate to comment on the forum about why you dislike certain features (or if there is something else you'd like me to implement). I appreciate all opinions/criticisms as it helps me make the mod even better!

Thank you!
